nautobot_ssot_infoblox/diffsync/adapters/infoblox.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, *args, job=None, sync=None, **kwargs):
         super().__init__(*args, **kwargs)
         self.job = job
         self.sync = sync
-        self.conn = InfobloxApi()
+        self.conn = conn
```

Comment:
Do we need the same error check as in InfobloxAdapter, i.e. `if self.conn in [None, False]: ... raise PluginImproperlyConfigured`?

Reply:
Good call. Added in.